OpenRA.Game/Map/MapCache.cs

```diff
@@ -39,6 +39,15 @@ public sealed class MapCache : IEnumerable<MapPreview>, IDisposable
 
 		public Dictionary<string, string> StringPool { get; } = new Dictionary<string, string>();
 
+		readonly List<MapDirectoryTracker> mapDirectoryTrackers = new List<MapDirectoryTracker>();
+		public string LastUpdatedUid;
+
+		/// <summary>
+		/// Called when a map has been updated. Provides old and new uid's of the map
+		/// Not called when map is added or deleted
+		/// </summary>
+		public event Action<string, string> MapUpdated = (oldUid, newuid) => { };
```

Comment:
Nitpicking: Use the casing of `UID` in the comment, and `newUid` in the parameter.

Reply:
fixed